FastSurferCNN/README.md

```diff
@@ -10,66 +10,77 @@ The network was trained with conformed images (UCHAR, 1-0.7 mm voxels and standa
+* `--t1`: name of the T1-weighted MRI_volume (like mri_volume.mgz, __default: orig.mgz__)
+* `--conformed_name`: name of the conformed MRI_volume (the input volume is always first conformed, if not already, and the result is saved under the given name, __default: orig.mgz__)
+* `--t`: search tag limits processing to subjects matching the pattern (e.g. sub-* or 1030*...)
+* `--sd`: Path to output directory (where should predictions be saved). Will be created if it does not already exist.
+* `--seg_log`: name of log-file (information about processing is stored here; If not set, logs will not be saved). Saved in the same directory as the predictions.
+* `--strip`: strip suffix from path definition of input file to yield correct subject name. (Optional, if full path is defined for `--t1`)
+* `--lut`: FreeSurfer-style Color Lookup Table with labels to use in final prediction. Default: ./config/FastSurfer_ColorLUT.tsv
+* `--seg`: Name of intermediate DL-based segmentation file (similar to aparc+aseg).
+* `--cfg_cor`: Path to the coronal config file
+* `--cfg_sag`: Path to the axial config file
+* `--cfg_ax`: Path to the sagittal config file
 
 #### Checkpoints
-* --ckpt_sag: path to sagittal network checkpoint
-* --ckpt_cor: path to coronal network checkpoint
-* --ckpt_ax: path to axial network checkpoint
+* `--ckpt_sag`: path to sagittal network checkpoint
+* `--ckpt_cor`: path to coronal network checkpoint
+* `--ckpt_ax`: path to axial network checkpoint
 
 #### Optional commands
-* --clean: clean up segmentation after running it (optional)
-* --run_viewagg_on: Define where the view aggregation should be run on. 
+* `--clean`: clean up segmentation after running it (optional)
+* `--viewagg_device`: Define where the view aggregation should be run on. 
                     By default, the program checks if you have enough memory to run the view aggregation on the gpu. 
                     The total memory is considered for this decision. 
-                    If this fails, or you actively overwrote the check with setting "--run_viewagg_on cpu", view agg is run on the cpu. 
-                    Equivalently, if you define "--run_viewagg_on gpu", view agg will be run on the gpu (no memory check will be done).
-* --device <str>: Device for processing ("cpu","cuda") (optional, default: cuda)
-* --hires: Switch on high resolution processing (no conforming to 1mm, but to smallest voxel size).
-* --batch_size: Batch size for inference. Default=8
+                    If this fails, or you actively overwrote the check with setting `--viewagg_device cpu`, view agg is run on the cpu. 
+                    Equivalently, if you define `--viewagg_device gpu`, view agg will be run on the gpu (no memory check will be done).
+* `--device <str>`: Device for processing ("cpu","cuda") (optional, default: cuda)
+* `--hires`: Switch on high resolution processing (no conforming to 1mm, but to smallest voxel size).
```

Comment:
I dont think this flag still exists

Reply:
This is resolved. See [here](https://github.com/Deep-MI/FastSurfer/pull/231)